PubNubSwiftConsole/Classes/UI/CollectionView/Cells/PresenceEventCollectionViewCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  PresenceEventCollectionViewCell.swift
+//  Pods
+//
+//  Created by Keith Martin on 8/10/16.
+//
+//
+
+import UIKit
+import PubNub
+
+protocol PresenceEventItem: Item {
+    init(itemType: ItemType, event: PNPresenceEventResult)
+    var eventType: String {get}
+    var occupancy: NSNumber? {get}
+    var timeToken: NSNumber? {get}
+}
+
+extension PresenceEventItem {
+    var title: String {
+        return eventType
+    }
+}
+
+struct Presence: PresenceEventItem {
```

Comment:
let's call this `PresenceEvent` because eventually there will be here now presence, and where now presence, and lack of description could lead to confusion